parsers/parse_FluidModelFromGid.m

```diff
@@ -232,6 +240,7 @@
 end
 out = cell2mat(out);
 fldMsh.nodes = out(:,2:4);
+fldMsh.nodeIDs = out(:,1);
```

Comment:
Why do we need the node IDs from GiD? I guess that is because when we have both structural and fluid nodes in the GiD input file then the nodes of the fluid problem don't start from 1, right? I guess in that case we will have a problem running the CFD case itself as the code in the current state assumes that the nodes are sequentially numbered starting from 1, right?